src/Models/MakeModel.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require_once 'src/Entities/Car.php';
+class MakeModel
+{
+    public PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function addMake(string $make_name) : bool
+    {   //add make
+        $query = $this->db->prepare("INSERT into `make`(`name`) VALUES (:inputmake);");
+        $query->bindParam(':inputmake', $make_name);
+        return $query->execute();
+        //would like to crossreference makes with a list of actual makes, or make a database of cars for the user to select from.
+    }
+    
+    public function getIdByName(string $make_name) : int
+    {
+        $query = $this->db->prepare("SELECT `id` FROM `make` WHERE `make`.`name` = :inputmake");
+        $query->bindParam(':inputmake', $make_name);
+        $query->execute();
+        $res = $query->fetch();
```

Comment:
What if this fails? If you put in a name that doesn't exist, $res will contain false